fwbot/views/fwbot_menu.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Encoding is preferred to be uppercase, i.e. `UTF-8`

Same in all other XML files.